sound_publish_rate: 1.0
volume_publish_rate: 1.0
sound_user_publish_rate: 1.0

error_sound_delay: 60

default_volume: 50
min_volume: 0
max_volume: 100

path_user_sound: "niryo_robot_user_sounds"
path_robot_sound: "niryo_robot_state_sounds"

robot_sounds:
  error_sound: "error4.wav"
  turn_on_sound: "start.wav"
  turn_off_sound: "stop.wav"
